packages/components/src/abbreviated-card/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { Card, CardBody, Icon } from '@wordpress/components';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../link';
+
+const AbbreviatedCard = ( { children, className, href, icon, onClick } ) => {
+	return (
+		<Card
+			className={ classnames(
+				'woocommerce-abbreviated-card',
+				className
+			) }
+		>
+			<CardBody size={ null }>
+				<Link href={ href } onClick={ onClick } type="wp-admin">
```

Comment:
Should we be hardcoding these `Link` types to `wp-admin`? If we're on an Analytics page they could be `wc-admin` to avoid a full pageload.

Reply:
Good suggestion! I modified this in the commit e4e7bdb and 36202c0